specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2151,16 +2378,16 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/partners": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/assemblies": {
       "get": {
         "tags": [
-          "IntegrationAccountPartners"
+          "IntegrationAccountAssemblies"
         ],
-        "operationId": "Partners_ListByIntegrationAccounts",
-        "description": "Gets a list of integration account partners.",
+        "operationId": "IntegrationAccountAssemblies_List",
+        "description": "List the assemblies for an integration account.",
         "x-ms-examples": {
```

Comment:
This should probably be an x-ms-pageable operation?

Reply:
Isn't a "top" query param required for that to work right? If that isn't required, I'll add the x-ms-pageable

Reply:
Nope - there are no mandatory query parameters for pageable. It only dictates how the generator interprets the response. Some services have additional query parameters to customize the paging behaviour. But this isn't necessary.